### DISCUSSION TITLE
text cursor in Curves section invisible

### DISCUSSION THREAD

**Discussion by liamauve** [Rem0o/FanControl.Releases#850] [General]
dark theme black primary color, the text cursor that flashes when you click on a name, is invisible. for comparison, in the controls section that cursor is white.

the highlighted text color is also not very visible, may i suggest changing it to the secondary color. 

cheers

**Comment by Rem0o** [comment]
Good comment.

I neglected that aspect a bit.  Will make use of the accent color in all the fields when selected.

Like this:

![image](https://user-images.githubusercontent.com/14118956/172015491-9ff954ce-ebf6-4ad3-90d7-96a0f1164196.png)
![image](https://user-images.githubusercontent.com/14118956/172015519-9f571555-3e57-4825-92b0-476a66a25630.png)

**Reply by Rem0o** [reply]
See V118
